src/box/lua/console.h

```diff
@@ -34,6 +34,8 @@
 extern "C" {
 #endif /* defined(__cplusplus) */
 
+typedef struct ev_loop ev_loop;
+struct ev_signal;
```

Comment:
This is unused. Looks like its left from the previous version. Please remove.

And please do some self-review before submitting a patch.

Reply:
Sorry, fixed.